### PR TITLE
Zoom sdk update

### PR DESCRIPTION
**Pull request by NavayuvanSB**
### Changes done
- Update zoom sdk to latest version

### Reason for the changes
- Previously integrated zoom sdk version is depreciated

### Guidelines
- [x] Have you self reviewed this PR in context to the previous PR feedbacks?

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -21,8 +21,9 @@ dependencies {
     implementation "ru.tinkoff.scrollingpagerindicator:scrollingpagerindicator:1.1.0"
 
     implementation 'com.github.kevinsawicki:wishlist:0.9@aar'
-//    implementation project(':commonlib')
-//    implementation project(':mobilertc')
+    // implementation project(':commonlib')
+    // implementation project(':mobilertc')
+
```
Comment (Karthik-0) [inline app/build.gradle:26]: Please undo this change

File: app/build.gradle [hunk 2]
```diff
@@ -41,8 +42,11 @@ dependencies {
     implementation 'de.greenrobot:greendao:2.0.0'
     
     // Testpress Course(which inbuilt has exam & core module)
-    implementation 'com.github.testpress.android-sdk:course:' + testpressSDK
-    implementation 'com.github.testpress.android-sdk:store:' + testpressSDK
+     implementation 'com.github.testpress.android-sdk:course:' + testpressSDK
+     implementation 'com.github.testpress.android-sdk:store:' + testpressSDK
+
+    // Testpress Course (for local sdk configuration)
+    // implementation 'in.testpress:course:1.0.0'
```
Comment (Karthik-0) [inline app/build.gradle:49]: Please undo these changes

File: build.gradle [hunk 3]
```diff
@@ -18,6 +18,11 @@ buildscript {
 
 allprojects {
     repositories {
+
+        // For local sdk configuration
+        // maven { url "/home/hp/.m2/repository/" }
+        // maven { url "https://phonepe.bintray.com/PhonePeIntentSDK" }
+
```
Comment (Karthik-0) [inline build.gradle:25]: This file should not be added as this change is not necessary

**Review by Karthik-0 — CHANGES_REQUESTED** [review]

**Comment by NavayuvanSB** [comment]
Comments resolved @Karthik-0, please review it.

**Comment by NavayuvanSB** [comment]
Refer #180 for further progress.

**Pull request closed without merging**